src/test/java/no/unit/nva/fileupload/AbortMultipartUploadHandlerTest.java

```diff
@@ -0,0 +1,108 @@
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import no.unit.nva.fileupload.model.AbortMultipartUploadRequestBody;
+import no.unit.nva.fileupload.model.SimpleMessageResponse;
+import no.unit.nva.fileupload.util.S3Constants;
+import no.unit.nva.testutils.HandlerUtils;
+import no.unit.nva.testutils.TestContext;
+import nva.commons.handlers.ApiGatewayHandler;
+import nva.commons.handlers.GatewayResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings({"checkstyle:javadoctype", "checkstyle:MissingJavadocMethod"})
+public class AbortMultipartUploadHandlerTest {
+
+    public static final String TEST_BUCKET_NAME = "bucketName";
+    public static final String SAMPLE_UPLOAD_ID = "uploadId";
+    public static final String SAMPLE_KEY = "key";
+    public static final String WILDCARD = "*";
+
+    private nva.commons.utils.Environment environment;
+    private AbortMultipartUploadHandler abortMultipartUploadHandler;
+    private HandlerUtils handlerUtils;
+    private ByteArrayOutputStream outputStream;
+    private Context context;
+    private AmazonS3Client s3client;
+
+    /**
+     * Setup test env.
+     */
+    @Before
+    public void setUp() {
+        environment = mock(nva.commons.utils.Environment.class);
+        when(environment.readEnv(ApiGatewayHandler.ALLOWED_ORIGIN_ENV)).thenReturn(WILDCARD);
+        when(environment.readEnv(S3Constants.S3_UPLOAD_BUCKET_KEY)).thenReturn(S3Constants.S3_UPLOAD_BUCKET_KEY);
+        s3client = mock(AmazonS3Client.class);
+        abortMultipartUploadHandler = new AbortMultipartUploadHandler(environment, s3client, TEST_BUCKET_NAME);
+        context = new TestContext();
+        handlerUtils = new HandlerUtils(objectMapper);
+        outputStream = new ByteArrayOutputStream();
+    }
+
+    @Test
+    public void testHandleRequestMissingParameters() throws IOException {
+        InputStream inputStream = handlerUtils
+                .requestObjectToApiGatewayRequestInputSteam(null, null);
+        abortMultipartUploadHandler.handleRequest(inputStream, outputStream, context);
+
+        GatewayResponse<SimpleMessageResponse> response = objectMapper.readValue(
+                outputStream.toByteArray(),
+                GatewayResponse.class);
+        assertEquals(SC_BAD_REQUEST, response.getStatusCode());
+    }
+
+    @Test
+    public void testHandleRequest() throws IOException {
```

Comment:
this is basically 2 tests. One for the code and one for the body. Renaming is required